pressbooks/pressbooks
# Add margin to blavatar span

Fix for #2276. This PR adds margin rules to the 'blavatar' `<span>` element Pressbooks uses to display the 'site icon' in the My Books menu, since the pre-existing margin styling was altered/removed in WordPress 5.8 (https://core.trac.wordpress.org/changeset/50834)

**To test:**
1. Switch to this branch
2. Observe your book list from the My Books menu in the admin dashboard. Check positioning of the PB logo and confirm that spacing has been improved (see screenshot below)
![Screenshot from 2021-07-27 11-00-40](https://user-images.githubusercontent.com/13485451/127204705-bc8ac29c-3853-4aba-8ac9-2b034120af20.png)
